eng/pipelines/runtimelab.yml

```diff
@@ -43,9 +43,9 @@ resources:
   containers:
   # Define auxilary containers used by the installer jobs
   - container: debpkg
-    image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-debpkg-20220504035737-cfdd435
+    image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-debpkg
   - container: rpmpkg
-    image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-rpmpkg-20210714125410-daa5116
+    image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-rpmpkg
```

Comment:
Particular reason to use different images?

Reply:
Not really, I just took what was in runtime.  I'll revert it.